Svc/Deframer/docs/sdd.md

```diff
@@ -386,27 +392,103 @@ Vertical dashed lines represent component code.
 Solid horizontal arrows represent synchronous port invocations, and open
 horizontal arrows represent asynchronous port invocations.
 
-![Active byte stream driver, command packet](img/sequence-diagrams/active-cmd-packet.png)
+```mermaid
+sequenceDiagram
+    activate activeComm
+    activeComm->>buffMgr: Allocate frame buffer FB
+    buffMgr-->>activeComm: Return FB
+    activeComm->>activeComm: Fill FB with framed data
+    activeComm->>deframer: Send FB[framedIn]
+    deframer->>buffMgr: Allocate packet buffer PB [bufferAllocate]
+    buffMgr-->>deframer: Return PB
+    deframer->>deframer: Deframe FB into PB
+    deframer->>deframer: Copy PB into a command packet C
+    deframer-)cmdDisp: Send C [comOut]
+    deframer->>buffMgr: Deallocation PB [bufferDeallocate]
```

Comment:
I think this should say `Deallocate PB` and similarly two lines below.

Reply:
Fixed